brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/VIP/Coordinator.swift

```diff
@@ -166,10 +166,22 @@ class BaseCoordinator: NSObject,
     }
     
     func showDeleteProfileInfo(keyMaster: KeyStore) {
-        openModally(coordinator: DeleteProfileInfoCoordinator.self, scene: Scenes.DeleteProfileInfo) { vc in
-            vc?.dataStore?.keyMaster = keyMaster
-            vc?.prepareData()
+        let nvc = RootNavigationController()
+        let coordinator = DeleteProfileInfoCoordinator(navigationController: nvc)
+        coordinator.start()
+        coordinator.parentCoordinator = self
+        
+        if let vc = coordinator.navigationController.children.first(where: { $0 is DeleteProfileInfoViewController }) as? DeleteProfileInfoViewController {
```

Comment:
considering u have just started the deleter coordinator, why do u check if the deleterVC is in the stach?